CIP-1694/README.md

```diff
@@ -0,0 +1,1124 @@
+
+Special provisions will apply in the initial bootstrap phase.
+Firstly, during the bootstrap phase, a quorum vote from the constitutional committee
+is sufficient to change the protocol parameters.
+Secondly, during the bootstrap phase, a quorum vote from the constitutional committee,
+together with a sufficient SPO vote, is sufficient to initiate a hard fork.
+No other actions are possible during the bootstrap phase.
+
+The bootstrap phase ends when **either** of the following occurs:
+1. a sufficient amount of stake is delegated to DReps
+2. a given number of epochs has passed (likely to be a number of months after the hard fork)
+
+#### Final safety measure, post bootstrapping
+
+Many people have stated that they believe that the actual voting turnout will not be so large
+as to be a strain on the throughput of the system.
+We also believe that this is likely to be the case, but when the bootstrap phase ends we will
+put one final, temporary safety	measure in place (this will also allow us to justify a low DRep deposit amount).
+
+For values of $X$ and $Y$ that are still to be determined,
+as soon as the bootstrap phase has ended,
+when we calculate the DReps stake distribution for the next epoch boundary,
+we will consider _only_ those DReps that are _either_ in the top $X$-many DReps ranked by stake amount,
+or those DReps that have at least $Y$ Lovelace.
+Every epoch, the value of $X$ will _increase_ and the value of $Y$ will decrease,
+so that eventually $X$ will be effectively infinite and $Y$ will be zero.
+Note that this is only an incentive, and nothing actually stops any DRep from casting their
+vote (though it will not be counted if it does not meet the requirements).
+
+If the community decides at some point that there is indeed a problem with congestion,
+then a hard fork could be enacted that limits the number of DReps in a more restrictive way.
+
+Reasonable numbers for the initial value of $X$ are probably 5,000-10,000.
+Reasonable numbers for the initial value of $Y$ are probably the total number of Lovelace
+divided by the initial value of $X$.
+
+The mechanism should be set to relax at a rate where the restriction is completely eliminated after
+a period of six months to one year.
+
+#### Other Ideas / Open Questions
+
+##### Pledge-weighted SPO voting
+
+The SPO vote could additionally be weighted by each SPO's pledge.
+This would provide a mechanism for allowing those with literal stake in the game to have a stronger vote.
+The weighting should be carefully chosen.
+
+##### Automatic re-delegation of DReps
+
+A DRep could optionally list another DRep credential in their registration certificate.
+Upon retirement, all of the DRep's delegations would be automatically transferred to the
+given DRep credential.  If that DRep had already retired, the delegation would be transfer
+to the 'Abstain' DRep.
+
+##### No DRep registration
```

Comment:
I really think we should stick with the dRep registration approach, it isn't any more  bureaucratic than delegating stake as a casual user, it's a process. It also defines the value of the dRep very well in the system.